test/CMakeLists.txt

```diff
@@ -73,21 +73,60 @@ endif()
 
 find_package(Threads)
 
-set(XTENSOR_IO_TESTS
-    main.cpp
-    test_xgdal.cpp
-    test_xnpz.cpp
-    test_xhighfive.cpp
-    test_ximage.cpp
-    test_xaudio.cpp
-    test_xio_blosc.cpp
-    test_xio_gzip.cpp
-    test_xio_zlib.cpp
-    test_xio_binary.cpp
-    test_xio_gcs_handler.cpp
-    test_xio_aws_handler.cpp
-    test_xio_gdal_handler.cpp
-)
+set(XTENSOR_IO_TESTS main.cpp test_xio_binary.cpp)
+set(XTENSOR_IO_FILES)
```

Comment:
I'm not sure if this is strictly necessary. Does CMake magically materialize variables on first use, so that `XTENSOR_IO_FILES` will pop into existence if any of the following conditionals are `TRUE`?